pkg/service/backup_backend_local.go

```diff
@@ -163,29 +164,32 @@ func toBackupDetails(e fs.DirEntry, prefix string) model.BackupDetails {
 	dirInfo, err := e.Info()
 	if err == nil {
 		details.LastModified = util.Ptr(dirInfo.ModTime())
-		details.Size = util.Ptr(dirEntrySize(prefix, e, dirInfo))
+		details.Size = folderSize(prefix)
```

Comment:
You can take it out of the if statement, since it is not related to the `dirInfo` anymore.